pwndbg/commands/context.py

```diff
@@ -554,16 +554,15 @@ def regs(regs=[]) -> None:
 
 
 def get_regs(regs: List[str] = None):
-    result = []
-
     if regs is None:
         regs = []
 
-    if len(regs) == 0:
+    elif len(regs) == 0:
```

Comment:
I don't think this is correct. If `regs` is None, we want to enter this if statement too, but if written like this that won't happen and this function won't print anything.

Reply:
Ugh, you are right. You know what.. let's just close it :D.